app/config/config.yml

```diff
@@ -23,24 +23,24 @@ csa_guzzle:
         elife_api:
             config:
                 base_uri: '%api_url%'
-                connect_timeout: '%api_connect_timeout%'
+#                connect_timeout: '%api_connect_timeout%'
```

Comment:
Ahem.

(Are you still having timeout problems in dev? It's 5 seconds now, so should be ok unless you have connection problems.)

Reply:
Gah!

(It's been patchy, but when it was timing out it seemed to be under doing so under 5s locally. From memory when it was timing out, it seemed about 1-2s between initiating page load to seeing a timeout message. Will keep an eye on it.)